README.md

```diff
@@ -1 +1,147 @@
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [hello-world-microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** is a starting project to implement Linters and to create new branches.
+
+## 🛠 Built With <a name="built-with">VS CODE EDITOR</a>
+
+### Tech Stack <a name="tech-stack">HTML & CSS</a> 
+  <details>
+    <summary>Client</summary>
+    <ul>
+      <li>HTML</li>
+      <li>CSS</li>
+    </ul>
+  </details>
+
+### Key Features <a name="key-features"></a> 
+
+Displays Hello-Microverse text on web page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to clone github repository using command : git clone https://github.com/PremChand12/hello-world-microverse
+
+### Setup
+
+Clone this repository to your desired folder: cd my-folder
+
+### Install
+
+Install this project with: cd my-project
+
+### Usage
+
+To run the project, use code editor e.g Visual Studio
+
+### Run tests
+
+To run tests, use Visual Studio Live Server
+
+### Deployment
+
+You can deploy this project using commands:
+git add,
+git commit and,
+git push
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Prem Chand**
+
+- GitHub: [@githubhandle](https://github.com/PremChand12)
+- Twitter: [@twitterhandle](https://twitter.com/Chandu1111112)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/prem-chand-maddati-15a69a1a8/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[add_buttons_to_website]**
+- [ ] **[add_gift_voucher_to_home_page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page].(https://github.com/PremChand12/hello-world-microverse/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please support the project by adding your contributions.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I want to thank my coding partner Nouman Walli who discussed the technical areas with me.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project has no license.
+
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly add the MIT license link to your readme file as this serves as a copywrite that gives the visitor of your page access to make use of the content you have on your page.
Add the MIT file to your root directory and target it in your readme file